s3downloader.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import boto3
+import json
+
+log = logging.getLogger('s3_downloader')
+log.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+# create formatter and add it to the handlers
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+# add the handlers to the logger
+log.addHandler(ch)
+
+endpoint_url = os.environ.get('S3_ENDPOINT_URL', 'http://localhost:9000')
+aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID', '')
+aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY', '')
+bucket = os.environ.get('S3_BUCKET', '')
+object_key = os.environ.get('S3_OBJECT_KEY', 'mapproxy.yaml')
```

Comment:
check if ssl_enabled is needed with @AnguIar

Reply:
fixed in 8090fbb